docs/writing_review_screen.md

```diff
@@ -0,0 +1,240 @@
+sidebar_label: |
+  Review Screen
+slug: /review_screen
+---
+
+## Overview
+
+Writing a review screen is very important to docassemble interviews. As the "back" button undoes an interview's progress,
+it's critical that people can edit answers they made early in the interview, without having to re-enter things later.
+
+Review screens are also critical for interviews to be fully WCAG compliant; for legal applications like the AssemblyLine was born from,
+[WCAG 2.1 success criterion 3.3.4](https://www.w3.org/TR/WCAG21/#error-prevention-legal-financial-data) requires that
+
+> a mechanism is available for reviewing, confirming, and correcting information before finalizing the submission
+
+in short, requiring a review screen to be present at the end of your interview, that allows the user to correct the information they entered.
+
+### Step 0: Complete the rest of the interview!
+
+Large changes to an interview's logic will require large changes to the review screen as well. Therefore, it's best to have mostly completed
+your interview before starting to make the review screen.
+
+### Step 1: ALDashboard Review Screen Generator
+
+Download the primary YAML files of your interview. You can download the YAML file either:
+
+* from the [docassemble playground](github#go-to-the-playground), clicking on the "Download" button below the YAML editor, or
+* from GitHub. For example, for the [Motion to Stay Eviction](https://github.com/SuffolkLITLab/docassemble-MotionToStayEviction/), you can navigate
+  to [the primary YAML file, SP6A.yml](https://github.com/SuffolkLITLab/docassemble-MotionToStayEviction/blob/main/docassemble/MotionToStayEviction/data/questions/SP6A.yml), and click the "download raw file" button.
+
+  ![A github page screenshot, the "download raw file" button in the top bar of the github file is circled](../docs/assets/review_download_yaml_from_github.png)
+
+You should download all of the YAML files where you define variables used in your interview. The more YAML files you download, the more content you might have to
+cut from the generated review screen.
+
+Once you have the YAML files, go to [the ALDashboard's review screen generator](https://apps-dev.suffolklitlab.org/start/ALDashboard/review_screen_generator), and upload your YAML files. You should leave the two checkboxes ("create revisit screens", and "point section blocks to review screen") checked. The next screen will have your review blocks! Make a file called `review.yml` in your playground, and copy the content from the review screen generator into it. Then, add a block in your main interview near the end like this:
+
+```yaml
+include:
+  - review.yml
+```
+
+### Step 2: Understanding the Generated Review Screen
+
+A first quick note is that the generated YAML might look different than how you're used to writing YAML files. The generated YAML uses lots of `|-`,
+which keeps the full text after, but removes the last newline. In short, something like this:
```

Comment:
I open with it, because the examples that I use in here are the concise versions, and this is explain to readers that these things are the same. IMO this core YAML syntax is stuff that is assumed in the docassemble docs and not covered well elsewhere, so it's important to have little tidbits like this.

I shortened it up a lot in this edit and I think the shorter version in an admonition is a good compromise.